picard/webservice/api_helpers.py

```diff
@@ -59,91 +58,74 @@ def _wrap_xml_metadata(data):
 
 
 class APIHelper(object):
+    base_path = "/"
 
-    def __init__(self, host, port, api_path, webservice):
-        self._host = host
-        self._port = port
-        self.api_path = api_path
+    def __init__(self, webservice):
```

Comment:
I'd make the base_path a parameter here. Actually I'd consider making it the base URL. Then you can easily initialize an API handler as e.g.

```python
lb_api = APIHelper(ws, base_url="https://api.listenbrainz.org/1/")
lb_api.get(....)
```

Derived sub classes can still be made easily by overriding `__init__`  and providing hard coded defaults.

Essentially this is also why tests break, as the tests test the APIHelper class in standalone use, which this PR currently breaks.